web/src/pages/news/podcasts/index.page.tsx

```diff
@@ -0,0 +1,132 @@
+import { NextSeo } from "next-seo";
+import React from "react";
+import { StoryblokStory } from "storyblok-generate-ts";
+
+import { ActionBanner } from "@nice-digital/nds-action-banner";
+import { Breadcrumbs, Breadcrumb } from "@nice-digital/nds-breadcrumbs";
+import { Button } from "@nice-digital/nds-button";
+import { Grid, GridItem } from "@nice-digital/nds-grid";
+import { PageHeader } from "@nice-digital/nds-page-header";
+import { Panel } from "@nice-digital/nds-panel";
+
+import { ErrorPageContent } from "@/components/ErrorPageContent/ErrorPageContent";
+import { NewsList } from "@/components/Storyblok/News/NewsList/NewsList";
+import { NewsListNav } from "@/components/Storyblok/News/NewsListNav/NewsListNav";
+import { NewsListPagination } from "@/components/Storyblok/News/NewsListPagination/NewsListPagination";
+import { NewsStory } from "@/types/News";
+import { validateRouteParams } from "@/utils/storyblok";
+
+import type { GetServerSidePropsContext } from "next";
+
+export type PodcastPostsProps = {
+	stories: StoryblokStory<NewsStory>[];
+	total: number;
+	currentPage: number;
+	perPage: number;
+	error?: string | undefined;
+};
+
+export const PodcastIndexPage = ({
+	stories,
+	currentPage,
+	total,
+	perPage,
+	error,
+}: PodcastPostsProps): React.ReactElement => {
+	if (error) {
+		return <ErrorPageContent title="Error" heading={error} />;
+	}
+	return (
+		<>
+			<NextSeo title="Podcasts" openGraph={{ title: "Podcasts" }}></NextSeo>
+			<PageHeader
+				heading="Podcasts"
+				variant="fullWidthDark"
+				lead="Our NICE talks podcasts bring you the real life experience of people working within NHS, public health and social care."
+				breadcrumbs={
+					<Breadcrumbs>
+						<Breadcrumb to="https://www.nice.org.uk/">Home</Breadcrumb>
```

Comment:
Really minor thing - might be worth removing the live URL here (so just leading with a slash) so that the breadcrumbs will allow people to stay in the same environment when testing.